config/locales/en.yml

```diff
@@ -29,21 +29,6 @@
 
 en:
   app_name: 'Stanford Digital Repository'
-  activemodel:
```

Comment:
Prefer messages over explicit over Rails magic. The current state of these messages are evidence why Rails magic leads to cruft.